README.md

```diff
@@ -277,6 +277,20 @@ $ go-licenses check \
 Note that dependencies from the ignored packages are still resolved and checked.
 This flag makes effect to `check`, `report` and `save` commands.
 
+### Include testing packages
+
+Use the `--include_tests` global flag to include packages only imported by testing code.
+For example, those packages could be testing libraries/frameworks.
+Usually you would want to add this flag because if you're already analyzing the licenses of your dependencies,
+likely those packages also need to be taken into account.
+Example command:
+
+```shell
+go-licenses check --include_tests "github.com/google/go-licenses"
```

Comment:
Can you try github.com/google/go-licenses/... ?

Just checking the root package may not include all deps used. It seems more reasonable to check all pkgs.

Reply:
Makes sense. Probably this is also worth mentioning in either `### Global` or `### Check`?

Reply:
good point

Reply:
Created #162 accordingly.